api/src/main/java/com/nicknathanjustin/streamercontracts/users/UserServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package com.nicknathanjustin.streamercontracts.users;
+
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+import java.sql.Timestamp;
+
+@RequiredArgsConstructor
+public class UserServiceImpl implements UserService {
+
+    @NonNull final UserModelRepository userModelRepository;
+
+    @Override
+    public UserModel createUser(@NonNull String twitchUsername) {
+        Timestamp creationTimestamp = new Timestamp(System.currentTimeMillis());
```

Comment:
super minor, but i feel real strongly about using the final modifier everywhere its possible. I think it ereally helps convey the developers intent that this variable isnt intended to change and helps to minimize bugs. Strongly recommend putting final everywhere you can when coding on the backend.